docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,468 @@
+        last patched image: ---Not Available---
+        workflow type: continuouspatchv1
+```
+
+If scan is currently running and a patched image exists
+```sh
+image: import:dotnetapp-manual
+        scan status: Running
+        scan date: 2024-09-13 21:05:58.841962+00:00
+        scan task ID: dt21
+        patch status: ---Not Available---
+        patch date: ---Not Available---
+        patch task ID: ---Not Available---
+        last patched image: import:dotnetapp-manual-patched
+        workflow type: continuouspatchv1
+```
+
+If scan is currently running and NO patched image exists
+```sh
+image: import:dotnetapp-manual
+        scan status: Running
+        scan date: 2024-09-13 21:05:58.841962+00:00
+        scan task ID: dt21
+        patch status: ---Not Available---
+        patch date: ---Not Available---
+        patch task ID: ---Not Available---
+        last patched image: ---Not Available---
+        workflow type: continuouspatchv1
+```
+
+If patch is currently running and a patched image exists
+```sh
+image: import:dotnetapp-manual
+        scan status: Succeeded
+        scan date: 2024-09-13 21:05:58.841962+00:00
+        scan task ID: dt21
+        patch status: Running
+        patch date: 2024-09-13 21:07:32.841962+00:00
+        patch task ID: xyz2
+        last patched image: import:dotnetapp-manual-patched
+        workflow type: continuouspatchv1
+```
+
+If patch is currently running and NO patched image exists
+```sh
+image: import:dotnetapp-manual
+        scan status: Succeeded
+        scan date: 2024-09-13 21:05:58.841962+00:00
+        scan task ID: dt21
+        patch status: Running
+        patch date: 2024-09-13 21:07:32.841962+00:00
+        patch task ID: xyz2
+        last patched image: ---Not Available---
+        workflow type: continuouspatchv1
+```
+
+## Canceling Running Tasks
+
+Sometimes you may need to cancel tasks which are currently running or waiting to run. For this purpose, please run the following CLI command:
+```sh
+az acr supply-chain workflow cancel-run -r <registryname> -g <resourcegroup> --type <continuouspatchv1>
+```
+
+This command will cancel all CSSC tasks within the registry with a status of “Running”, “Queued” and “Started”. The command will output a success or failure. Failure results will follow the failure pattern of the other workflow commands if the input is incorrect.
+Running the cancel command will only affect tasks in the current schedule. For example, if a user has their schedule for 1d, and runs the cancel command, tasks in those 3 states will be canceled for that day, but will be requeued for the next day. If the schedule was a week, then that week’s tasks would be canceled, but the following week would have the tasks requeued. The common scenario for using this command is to cancel all running tasks in case a user misconfigures their continuous patching workflow and doesn’t want to wait for all tasks to finish running.
```

Comment:
I thought the `--schedule` option controls the number of days that the images will be scanned, but the images will always be scanned daily regardless?

You seem to imply that `--schedule` controls the cadence but I believe it controls the number of days in which the image will be scanned daily?

Please reword this section.

Reply:
Wait so what exactly does `--schedule` imply? When you first introduced `--schedule` way earlier in the doc - you also did not clarify what it exactly controls (the cadence? or the length in time?). It would be good to clarify it there earlier too.